pkgs/development/libraries/libtomcrypt/default.nix

```
@@ -17,14 +17,16 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  nativeBuildInputs = [ libtool ];
+  nativeBuildInputs = [ libtool libtommath ];
 
   postPatch = ''
     substituteInPlace makefile.shared --replace "LT:=glibtool" "LT:=libtool"
   '';
 
   preBuild = ''
     makeFlagsArray=(PREFIX=$out \
```

```suggestion
    makeFlagsArray+=(PREFIX=$out \
```

Or do we want to overwrite makeFlagsArray?

---

Done